File: TS/Typescript_Lab2/Department.js
"use strict";
exports.__esModule = true;
exports.printInForDepartment = exports.Department = void 0;
var Department = /** @class */ (function () {
    function Department(id, name) {
        this.id = id;
        this.name = name;
    }
    Department.prototype.getid = function () {
        return this.id;
    };
    Department.prototype.setid = function (id) {
        this.id = id;
    };
    Department.prototype.getname = function () {
        return this.name;
    };
    Department.prototype.setname = function (name) {
        this.name = name;
    };
    return Department;
}());
exports.Department = Department;
function printInForDepartment(department) {
    console.log("ID " + department.getid() + "name " + department.getname());
}
exports.printInForDepartment = printInForDepartment;

File: JavaAdvan/Hibernate_Lab2/src/main/java/com/vti/entity/Department.java
package com.vti.entity;

import java.io.Serializable;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Entity
@Table(name = "Department", catalog = "TestingSystemHibernate2")
@Setter
@Getter
@NoArgsConstructor
@ToString

public class Department implements Serializable {
	@Column(name = "DepartmentID")
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private short id;

	@Column(name = "DepartmentName", length = 30, nullable = false, unique = true)
	private String name;

//	@Override
//	public String toString() {
//		return "Department [id=" + id + ", name=" + name + "]";
//	}
//
//	public short getId() {
//		return id;
//	}
//
//	public void setId(short id) {
//		this.id = id;
//	}
//
//	public String getName() {
//		return name;
//	}
//
//	public void setName(String name) {
//		this.name = name;
//	}

}
